assignmentPR/MathFunc1.c

```diff
@@ -0,0 +1,316 @@
+/* 
+
+MathFunc01.c
+Written by Stephen White, Kalen Wood-Wardlow, & Gary Baker
+9/23/18
+
+*/
+
+/* Import Statements (no math allowed) */
+#include <stdlib.h>
+#include <stdio.h>
+
+
+/* Global Constants */
+typedef int bool;
+#define true 1
+#define false 0
+
+const int CHAR_ARRAY_SIZE_MAX = 80; 
+const int INT_CONVERSION_ERROR = -1;
+const int WHITESPACE = 32; 
+const int SAME_STRING = 0;
+
+
+/* Function Declarations */
+
+bool characteristic(char numString[], int *c);
+
+bool mantissa(char numString[], int *numerator, int *denominator);
+
+int compareString(const char *stringOne, const char *stringTwo);
+
+void cleanCharArray(char *charArray, int arraySize);
+
+int isNumericChar(char character);
+
+int myAtoi(char *stringSequence);
+
+void reverse(char string[]);
+
+int myStrLen(char *string);
+
+int tenToThePowerOf(int stringLength);
+
```

Comment:
This is what I mean when I say this is very modularized, it does make it quite easy to understand exactly what you guys are doing.